Block/Configuration.php

```diff
@@ -209,6 +211,36 @@ public function getConfiguration()
                 'addToCartSelector' => $config->getConversionAnalyticsAddToCartSelector(),
                 'orderedProductIds' => $this->getOrderedProductIds($config, $request),
             ],
+            'isPersonalizationEnabled' => $persoHelper->isPersoEnabled(),
+            'personalization' => [
+                'enabled' => $persoHelper->isPersoEnabled(),
+                'viewedEvents' => [
+                    'viewProduct' => [
+                        'eventName' => 'Viewed Product',
+                        'enabled' => $persoHelper->isViewProductTracked(),
+                        'method' => 'viewedObjectIDs',
+                    ],
+                ],
+                'clickedEvents' => [
+                    'productClicked' => [
+                        'eventName' => 'Product Clicked',
+                        'enabled' => $persoHelper->isProductClickedTracked(),
+                        'selector' => $persoHelper->getProductClickedSelector(),
+                        'method' => 'clickedObjectIDs',
+                    ],
+                    'productRecommended' => [
+                        'eventName' => 'Recommended Product Clicked',
+                        'enabled' => $persoHelper->isProductRecommendedTracked(),
+                        'selector' => $persoHelper->getProductRecommendedSelector(),
+                        'method' => 'clickedObjectIDs',
+                    ],
+                ],
+                'filterClicked' => [
+                    'eventName' => 'Filter Clicked',
+                    'enabled' => $persoHelper->isFilterClickedTracked(),
+                    'method' => 'clickedFilters',
```

Comment:
Are the the only events the user can send ?
For a pure person perspective it's best that the user can freely pick their event names. I understand there are tradeoff magento needs to make to help the setup of person but I'm curious on how easy it can be to add move event names

Reply:
You can configure the events for view and conversion options as well. The ones identified in the config object are for the click events. 

Regarding the ability to choose their event name, I find the ones that we supplied are very generic enough for the events we identified. I think one of the things we can do to improve this is to translate the event name so that they can be translated to their language or label of choice through Magento's translate feature. 

The requirements for this implementation didn't include a custom section to create other events to track other things. Perhaps this something we can revisit in P2.

Reply:
Yes definitely. I think the event set right now if very complete to benefit from all the features perso opens. Thanks for clarifying!